app/editor/src/features/content/form/ContentSummaryForm.tsx

```diff
@@ -180,119 +184,131 @@ export const ContentSummaryForm: React.FC<IContentSummaryFormProps> = ({
-          />
-        </Col>
-      </Row>
-      <Row className="textarea">
-        <Col flex="1 1 0">
-          <Show visible={contentType === ContentTypeName.Snippet}>
-            <FormikTextArea
-              name="summary"
-              label="Summary"
-              required
-              onBlur={(e) => {
-                const value = e.currentTarget.value;
-                if (!!value) {
-                  const values = value.match(tagMatch)?.toString()?.split(', ') ?? [];
-                  const tags = extractTags(values);
-                  setFieldValue('tags', tags);
-                }
-              }}
+        <Show visible={contentType !== ContentTypeName.Image}>
+          <Col className="licenses">
+            <RadioGroup
+              name="expireOptions"
+              label="License"
+              spaceUnderRadio
+              options={licenseOptions}
+              value={licenseOptions.find((e) => e.value === values?.licenseId)}
+              onChange={(e) => setFieldValue('licenseId', Number(e.target.value))}
             />
-          </Show>
-          <Show visible={contentType !== ContentTypeName.Snippet}>
-            <FormikTextArea
-              name="body"
-              label="Story"
-              required
-              onBlur={(e) => {
-                const value = e.currentTarget.value;
-                if (!!value) {
-                  const values = value.match(tagMatch)?.toString()?.split(', ') ?? [];
-                  const tags = extractTags(values);
-                  setFieldValue('tags', tags);
-                }
-              }}
-            />
-          </Show>
-        </Col>
+          </Col>
+        </Show>
       </Row>
-      <Row>
-        <FormikText
-          name="tags"
-          label="Tags"
-          disabled
-          width={combined ? FieldSize.Big : FieldSize.Large}
-          value={values.tags.map((t) => t.id).join(', ')}
-        />
-        <Button
-          variant={ButtonVariant.danger}
-          className="top-spacer"
-          onClick={() => {
-            const regex = /\[.*\]/; // TODO: This is far too eager and could remove valuable content.
-            setFieldValue('summary', values.summary.replace(regex, ''));
-            setFieldValue('tags', []);
-          }}
-        >
-          Clear Tags
-        </Button>
-      </Row>
-      <Row className="row-margins">
-        <FormikRadioGroup
-          label="Toning"
-          direction="row"
-          error={savePressed && toningError}
-          name="tonePool"
-          required
-          options={toningOptions}
-          onChange={(e, value) => {
-            setFieldValue('tonePool', value);
-            setFieldValue('tone', value?.value);
-          }}
-        />
-      </Row>
-      <Show visible={contentType === ContentTypeName.Snippet}>
+      <Show
+        visible={contentType === ContentTypeName.Snippet || contentType !== ContentTypeName.Image}
```

Comment:
Don't think this is correct

Reply:
Updated (the logic seems to be fine however it could be simplified by removing the first condition), thanks